docs/.vitepress/theme/Utils/fetch.ts

```diff
@@ -1,8 +1,8 @@
-export function universalFetch(url: string, headers: any) {
+import fetch from 'node-fetch'
```

Comment:
Having this as a global import might have undesirable consequences for browser builds. We should not bundle it together with the webpage, that's why a dynamic import was used previously